docs/en/guides/Plugin-test.md

```diff
@@ -485,6 +527,80 @@ SegmentB span list should like following
     - {parentEndpoint: /httpclient-case/case/httpclient, networkAddress: 'localhost:8080', refType: CrossProcess, parentSpanId: 1, parentTraceSegmentId: not null, parentServiceInstance: not null, parentService: not null, traceId: not null}
 ```
 
+### The example Process of Writing Meter Expected Data
+
+Expected data file, `expectedData.yaml`, include `MeterItems` part.
+
+We are using the toolkit plugin to show how to write the expected data. You could write the meter in the plugin, please follow [this document](Java-Plugin-Development-Guide.md#meter-plugin).
+
+There is one key point of testing
+1. Build a meter and operate it.
+
+Such as `Counter`:
+```java
+MeterFactory.counter("test_counter").tag("ck1", "cv1").build().increment(1d);
+MeterFactory.histogram("test_histogram").tag("hk1", "hv1").steps(1d, 5d, 10d).build().addValue(2d);
+```
+
+```
++-------------+         +------------------+
+|   Plugin    |         |    Agent core    |
+|             |         |                  |
++-----|-------+         +---------|--------+
+      |                           |         
+      |                           |         
+      |    Build or operate      +-+        
+      +------------------------> |-|        
+      |                          |-]
+      |                          |-|        
+      |                          |-|        
+      |                          |-|
+      |                          |-|        
+      | <--------------------------|        
+      |                          +-+        
+      |                           |         
+      |                           |         
+      |                           |         
+      |                           |         
+      +                           +         
+```
+
+#### meterItems
+
+By following the flow of the toolkit case,  there should be two meters created.
+1. Meter `test_counter` created from `MeterFactory#counter`. Let's name it as `MeterA`.
+1. Meter `test_histogram` created from `MeterFactory#histogram`. Let's name it as `MeterB`.
+
+```yml
+meterItems:
+  - serviceName: toolkit-case
+    meterSize: 2
+```
+
+They're showing two kinds of meter, MeterA has a single value, MeterB has a histogram value.
+
+MeterA should like following, `counter` and `gauge` use the same data format.
+```yaml
+- meterId:
+    name: test_counter
+    tags:
+      - {name: ck1, value: cv1}
+  singleValue: gt 0
+```
+
+MeterB should like following.
+```yaml
+- meterId:
+    name: test_histogram
+    tags:
+      - {name: hk1, value: hv1}
+  histogram:
+    - {bucket: 0.0, count: ge 0}
+    - {bucket: 1.0, count: ge 0}
+    - {bucket: 5.0, count: ge 0}
+    - {bucket: 10.0, count: ge 0}
```

Comment:
I have a little concern about the practicality. When the codes try to use the histogram to measure the performance, it seems hard to tell which bucket(s) would have value. Could you try to do a bucket check(w/o count) only, and only check at least not all `count`(s) are 0?

@dmsolr What do you think?

Reply:
Follow the https://github.com/apache/skywalking-agent-test-tool/pull/26, change the histogram meter validation way.